pyvista_examples/README.rst

```diff
@@ -0,0 +1,18 @@
+.. _pyvista-examples-index:
+
+Gallery of Examples using PyVista
+=================================
+
+
+.. _general_pyvista_examples:
+
+PyVista Examples
+----------------
+
+Examples from the Sphinx-Gallery using PyVista for embedding 3D plots.
+`Learn more about PyVista <https://docs.pyvista.org>`_ and see their
+`extensive examples gallery <https://docs.pyvista.org/examples/index.html>`_
+built with Sphinx-Gallery!
+
+Take a look at the :ref:`image_scrapers` section to learn more about using
+PyVista in your example gallery.
```

Comment:
I think it's worth mentioning explicitly that it is *not* enabled by default.

Reply:
Beat me to it, thanks!